src/PqHandle.php

```diff
@@ -294,6 +295,14 @@ private function release(): void
         $deferred->resolve();
     }
 
+    private function escapeParams(array &$params): void
+    {
+        foreach ($params as &$param) {
+            if ($param instanceof ByteA) {
+                $param = $this->handle->escapeByteA($param->getString());
+            }
+        }
+    }
```

Comment:
This should return a new array instead of using references.

Reply:
Done!